MerlinAU.sh

```diff
@@ -3221,6 +3241,66 @@ _DoUninstall_()
    _DoExit_ 0
 }
 
+##-------------------------------------##
+## Added by Martinski W. [2024-Feb-18] ##
+##-------------------------------------##
+_SetEMailFormatType_()
+{
+   local doReturnToMenu
+   local currFormatOpt  nextFormatOpt  currFormatStr
+
+   currFormatOpt="$(Get_Custom_Setting FW_New_Update_EMail_FormatType)"
+   if [ -z "$currFormatOpt" ] || [ "$currFormatOpt" = "TBD" ]
+   then
+       nextFormatOpt=""
+       currFormatOpt="$sendEMailFormaType"
+   else
+       nextFormatOpt="$currFormatOpt"
+   fi
+   currFormatStr="Current Format: ${GRNct}${currFormatOpt}${NOct}"
+
+   doReturnToMenu=false
+   while true
+   do
+       printf "\n${SEPstr}"
```

Comment:
Btw you did good with the layout of this menu, well done.
Kinda makes me want to steal it for the ROG option instead of just having a yes or no lol!